docs/client-libraries-producers.md

```diff
@@ -41,33 +52,45 @@ Producer<String> producer = pulsarClient.newProducer(Schema.STRING)
 </Tabs>
 ````
 
-## Send messages
+## Publish messages
+
+This example shows how to publish messages using producers. The publish operation will be done until the broker tells us that the message has been successfully published. It will return the message id after the message is published successfully.
```

Comment:
```suggestion
This example shows how to publish messages using producers. The publish operation is done until the broker tells you the message has been successfully published. Pulsar returns the message ID after the message is published successfully.
```

1. Who returns the msg ID? Pulsar? Need to specify
2. Writing style: 
a. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp
b. Avoid the first-person pronouns I and we https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.hyip4lzbapvm

Reply:
Thanks for your guidance. 

> Who returns the msg ID? Pulsar? Need to specify

It's the broker. I have fixed it.